applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.h

```diff
@@ -43,6 +43,15 @@ int broadcast_sink_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32
  */
 int broadcast_sink_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_id);
 
+/**
+ * @brief	Set the broadcast code for the Bluetooth LE Audio broadcast sink.
+ *
+ * @param[in]	broadcast_code	Pointer to the broadcast code.
+ *
+ * @return	0 for success, error otherwise.
```

Comment:
We should mention BT_ISO_BROADCAST_CODE_SIZE here, since there is no size parameter :)